tests/queries/0_stateless/01246_buffer_flush.sql

```diff
@@ -9,14 +9,14 @@ create table data_01256 as system.numbers Engine=Memory();
 
 select 'min';
 create table buffer_01256 as system.numbers Engine=Buffer(currentDatabase(), data_01256, 1,
-    2, 100, /* time */
+    5, 100, /* time */
     4, 100, /* rows */
     1, 1e6  /* bytes */
 );
 insert into buffer_01256 select * from system.numbers limit 5;
 select count() from data_01256;
--- sleep 2 (min time) + 1 (round up) + bias (1) = 4
-select sleepEachRow(2) from numbers(2) FORMAT Null;
+-- It is enough to ensure that the buffer will be flushed earlier then 2*min_time (10 sec)
```

Comment:
But it is not guaranteed whatsoever. The thread with buffer flush can be delayed for arbitrary time.

Reply:
It is time related test, there is no way to guarantee that the flush will happen, the only thing that came to my mind is to add `no-parallel` test, to reduce influence of other tests